src/gui/qgsmaptool.h

```diff
@@ -278,7 +278,7 @@ class GUI_EXPORT QgsMapTool : public QObject
      */
     QgsPoint toMapCoordinates( const QgsMapLayer *layer, const QgsPoint &point ) SIP_PYNAME( toMapCoordinatesV2 );
 
-    //! trnasformation of the rect from map coordinates to layer's coordinates
+    //! transformation of the rect from map coordinates to layer's coordinates
```

Comment:
```suggestion
    //! Transforms a \a rect from map coordinates to a \a layer's coordinates.
```

May as well fix the whole string up :smile:

Reply:
There are more strings in the same style in the file. I'll change them in a separate pull request to the format you suggested.

Reply:
I've opened #41216 for the class member documentation improvements.